bundles/org.eclipse.passage.lic.licenses.edit/src-gen/org/eclipse/passage/lic/licenses/edit/providers/EvaluationInstructionsItemProvider.java

```diff
@@ -103,11 +103,11 @@ protected void addExpressionPropertyDescriptor(Object object) {
 	 * This returns EvaluationInstructions.gif.
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
-	 * @generated
+	 * @generated not
 	 */
 	@Override
 	public Object getImage(Object object) {
-		return overlayImage(object, getResourceLocator().getImage("full/obj16/EvaluationInstructions")); //$NON-NLS-1$
+		return null;
```

Comment:
I think this is what super returns by default
Perhaps we should just tune the genmodel to omit images for this EClass

Reply:
All these elements are available in gui and must be represented by an image. We just do not have ones yet.